tests/Unit/ParallelAlgorithms/Actions/Test_SetData.cpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "ParallelAlgorithms/Actions/SetData.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+#include "tests/Unit/ActionTesting.hpp"
+
+namespace {
+
+struct SomeNumber : db::SimpleTag {
```

Comment:
SomeInt?

Reply:
It’s a tag used only for this test and I don’t think this name change makes the test any clearer, so I’ll keep it as-is unless you feel strongly about it for some reason :)